packages/components/src/components/tabs/tabs.jsx

```diff
@@ -24,13 +25,26 @@ class Tabs extends Component {
     }
 
     render() {
-        const { children }     = this.props;
+        const { children,
+            className,
+            top,
+            bottom,
+            fit_content }      = this.props;
         const { active_index } = this.state;
         const tab_width        = (100 / children.length).toFixed(2);
 
         return (
-            <div className='dc-tabs' style={{ '--tab-width': `${tab_width}%` }}>
-                <ul className='dc-tabs__list'>
+            <div
+                className={`dc-tabs dc-tabs--${className}`}
```

Comment:
is `className` always passed? if not I suggest:

```suggestion
                className={classNames(`dc-tabs', { 'dc-tabs--${className}`: className }}
```

Reply:
@negar-binary we only use class names where there is condition, otherwise the template literals is enough.

Reply:
won't it be `<div class='dc-tabs dc-tabs--'` for those that don't pass className? or everyone is passing it?